src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java

```diff
@@ -47,6 +49,8 @@ public ReportSender(
     }
     // endregion
 
+    @Scheduled(cron = "${reports.cron:-}")
```

Comment:
can we please remove the default value from here and configure the prop in app yaml. Makes it consistent with what we have in other places.